lib/elixometer.ex

```diff
@@ -348,8 +348,7 @@ defmodule Elixometer do
       subscribe_options = cfg[:subscribe_options] || []
 
       if reporter do
-        :exometer.info(metric_name)
-        |> Keyword.get(:datapoints)
+        :exometer.info(metric_name, :datapoints)
```

Comment:
@ianwalter that's really interesting; usually the two args info pulls out a single key, but in this case, he's not returning the datapoints for some reason. 

I'm using the following exometer/exometer_core
```elixir
"exometer": {:git, "https://github.com/PSPDFKit-labs/exometer.git", "fc367682a809ccd463a907e6e00e52d6d354c83f", []},
  "exometer_core": {:git, "https://github.com/PSPDFKit-labs/exometer_core.git", "ccc18cf0328692756fd9b837a88823cf979da8e1", []},
```

Which ones are you using?
FYI, this seems to work with mine, so I think I'm ok with the change when there are some tests.